File: beecrowd_urionlinejudge/uri1012Area.c
#include <stdio.h>
#include <math.h>

int main() {
double pi,atriangulo,acirculo,atrapezio,aquadrado,aretangulo,a,b,c;

scanf("%lf %lf %lf", &a,&b,&c);

pi = 3.14159f;
atriangulo = (a*c)/2.0f;
acirculo = pi*pow(c,2.0f);
atrapezio =  ((a+b)*c)/2.0f;
aquadrado = b*b;
aretangulo = a*b;

printf("TRIANGULO: %0.3lf\n",atriangulo);
printf("CIRCULO: %0.3lf\n",acirculo);
printf("TRAPEZIO: %0.3lf\n",atrapezio);
printf("QUADRADO: %0.3lf\n",aquadrado);
printf("RETANGULO: %0.3lf\n",aretangulo);

    
	return 0;
}

File: beecrowd_urionlinejudge/uri1012Area.cs
    using System;
    
    class URI {
    
        static void Main(string[] args) { 

double pi,atriangulo,acirculo,atrapezio,aquadrado,aretangulo,a,b,c;
    

			string[] texto = Console.ReadLine().Split(" ".ToCharArray());
            a = float.Parse(texto[0]);
			b = float.Parse(texto[1]);
			c = float.Parse(texto[2]);

pi = 3.14159;
atriangulo = (a*c)/2.0;
acirculo = pi*Math.Pow(c,2.0);
atrapezio =  ((a+b)*c)/2.0;
aquadrado = b*b;
aretangulo = a*b;



            System.Console.WriteLine(String.Format( "TRIANGULO: {0:0.000}",atriangulo));
            System.Console.WriteLine(String.Format( "CIRCULO: {0:0.000}",acirculo));
            System.Console.WriteLine(String.Format( "TRAPEZIO: {0:0.000}",atrapezio));
            System.Console.WriteLine(String.Format( "QUADRADO: {0:0.000}",aquadrado));
            System.Console.WriteLine(String.Format( "RETANGULO: {0:0.000}",aretangulo));

                
        }
    
    }



